openedx/core/djangoapps/catalog/tests/factories.py

```diff
@@ -6,7 +6,6 @@
 import uuid
 from faker import Faker
 
-
```

Comment:
This could cause an error with spacing from the import functions requiring two lines